applications/webapp/src/Pages/User/userPage.tsx

```diff
@@ -50,8 +73,9 @@ export default function UserPage() {
         {data && (
           <ProfileHolder
             aria-label="profile holder"
-            username={Username}
+            username={userDetail.username}
             description={mockUser.description}
+            editUser={editUser}
```

Comment:
See the other comment, but this is the same situation here. `editUser` is implemented in `UserPage` and then just passed down to `ProfileHolder`. Bad practice.